_sass/blocks/revenues/_revenues_subpage_intro.scss

```diff
@@ -20,10 +20,17 @@
 
 .revenues_subpage-office {
   background-color: $white;
-  border: $base-border-size solid $mid-gray;
-  border-radius: $base-border-radius;
-  margin-bottom: $base-padding;
-  padding-top: $base-padding;
+  &:first-child {
+    padding-top: $base-padding;
```

Comment:
Properties should be ordered border-top-left-radius, border-top-right-radius, padding-top